bindings/http/http.go

```diff
@@ -104,6 +107,13 @@ func (h *HTTPSource) Init(metadata bindings.Metadata) error {
 		h.errorIfNot2XX = true
 	}
 
+	if val, ok := metadata.Properties["traceHeaders"]; ok {
```

Comment:
Could be simplified to:

utils.IsTruthy(metadata.Properties["traceHeader"])

No need for the if block at all: if the value doesn't exist, the function returns false by default.

Reply:
Also please make the key a constant

Reply:
On second thought I don't think we need this metadata key. Seems that the headers need to be passed in the metadata of each request so the default behavior is to not include them anyways.

Reply:
Daprd is currently passing the trace headers to Bindings by adding them to metadata, but the HTTP binding is not adding them to the request. These trace headers exist when Dapr tracing is configured and regardless of whether the client calling Dapr includes them in metadata. And they have the advantage of getting the current Span id from Dapr (instead of the span id of the client). 

So the existing behavior will change for some. It seems like a minor change of behavior, but I wasn’t sure. Thoughts?

Reply:
Removed component metadata based on discussion below.